Frends.Facebook.Request/Frends.Facebook.Request.sln

```diff
@@ -3,9 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.0.32112.339
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Frends.Facebook.Get", "Frends.Facebook.Get\Frends.Facebook.Get.csproj", "{35C305C0-8108-4A98-BB1D-AFE5C926239E}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Frends.Facebook.Request", "Frends.Facebook.Request\Frends.Facebook.Request.csproj", "{35C305C0-8108-4A98-BB1D-AFE5C926239E}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Frends.Facebook.Get.Tests", "Frends.Facebook.Get.Tests\Frends.Facebook.Get.Tests.csproj", "{8CA92187-8E4F-4414-803B-EC899479022E}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Frends.Facebook.Request.Tests", "Frends.Facebook.Request.Tests\Frends.Facebook.Request.Tests.csproj", "{8CA92187-8E4F-4414-803B-EC899479022E}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{78F7F22E-6E20-4BCE-8362-0C558568B729}"
 	ProjectSection(SolutionItems) = preProject
```

Comment:
https://github.com/FrendsPlatform/Frends.Facebook/blob/4e503fd1b23e14c95357425ba363da9b131e75ec/Frends.Facebook.Request/Frends.Facebook.Request.sln#L13-L15
Please replace the correct workflows.